services/src/main/java/org/keycloak/services/DefaultKeycloakSessionFactory.java

```diff
@@ -18,25 +18,13 @@
 
 import org.jboss.logging.Logger;
 import org.keycloak.Config;
+import org.keycloak.common.Profile;
 import org.keycloak.common.util.MultivaluedHashMap;
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.models.KeycloakSessionFactory;
-import org.keycloak.provider.EnvironmentDependentProviderFactory;
-import org.keycloak.provider.Provider;
-import org.keycloak.provider.ProviderEvent;
-import org.keycloak.provider.ProviderEventListener;
-import org.keycloak.provider.ProviderFactory;
-import org.keycloak.provider.ProviderManager;
-import org.keycloak.provider.ProviderManagerDeployer;
-import org.keycloak.provider.ProviderManagerRegistry;
-import org.keycloak.provider.Spi;
-
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import org.keycloak.provider.*;
```

Comment:
yeah... just updated and started from a fresh IDEA instance.  Haven't taken the time to tune all the things back yet.  Fixed now.